rotkehlchen/api/v1/resources.py

```diff
@@ -3243,3 +3249,59 @@ def post(self, token: EvmToken) -> Response:
     @use_kwargs(post_delete_schema, location='json_and_query')
     def delete(self, token: EvmToken) -> Response:
         return self.rest_api.remove_token_from_spam(token=token)
+
+
+class CalendarResource(BaseMethodView):
+
+    create_schema = NewCalendarEntrySchema()
+    delete_schema = DeleteCalendarEntrySchema()
+    update_schema = UpdateCalendarSchema()
+    query_schema = QueryCalendarSchema()
+
+    @require_loggedin_user()
+    @use_kwargs(create_schema, location='json_and_query')
+    def put(
+            self,
+            name: str,
+            timestamp: int,
```

Comment:
same questions for all of these. Type of `Timestamp` and optionality of description